snapshot-labs/sx-monorepo
# fix: Update shutter logo

### Summary

- Update shutter logo with new one from https://www.shutter.network/
- Remove `fill="white"` on "shutter" keyword to make it work on light mode
- Remove "powered by" and make shutter logo look same everywhere

Active Proposal:
| Before    | After |
| -------- | ------- |
| <img width="335" alt="Untitled 5" src="https://github.com/user-attachments/assets/fcf6d701-31c1-4d7e-b27e-83ebb7153a19">  | <img width="340" alt="Untitled 6" src="https://github.com/user-attachments/assets/703ed56f-5afe-44ef-9d93-fed91dd802ea">    |
| <img width="343" alt="Untitled 8" src="https://github.com/user-attachments/assets/bff929ac-7fae-4fd2-b879-3af377198562"> |  <img width="343" alt="Untitled 7" src="https://github.com/user-attachments/assets/82ff64e6-31f2-4bbf-9cf1-6ac71f27301b"> |

Closed Proposal:
| Before    | After |
| -------- | ------- |
| <img width="345" alt="Untitled 11" src="https://github.com/user-attachments/assets/9d7e490d-995d-45f0-ab80-58b86f333499"> | <img width="344" alt="Untitled 12" src="https://github.com/user-attachments/assets/688d7227-1c57-4586-aa45-111168e5a862"> |
|  <img width="341" alt="Untitled 9" src="https://github.com/user-attachments/assets/b83219fe-632e-43d0-86f7-37634a24320b">  | <img width="340" alt="Untitled 10" src="https://github.com/user-attachments/assets/39b9d71d-c6fa-4eac-9e98-5bde8985dac4">  |

Closes https://github.com/snapshot-labs/workflow/issues/136

### How to test

1. Go to proposals:
http://localhost:8080/#/s:shutterdao0x36.eth/proposal/0xfbb3c3a53830903896bac3c74eab2c5b60909dabc3cc01ff2a4c08a27f262ded
http://localhost:8080/#/s:shutterdao0x36.eth/proposal/0x4bb5cf27998b7371d1c69f7d2d0e433d601bcb46e2a2cdd38108a743c432fa35
3. You should see an updated logo
